roles/etcd_migrate/tasks/check.yml

```diff
@@ -1,4 +1,8 @@
 ---
+- fail:
+    msg: "Currently etcd v3 migration is unsupported while we test it more thoroughly"
+  when: not openshift_enable_unsupported_configurations | default(false) | bool
```

Comment:
@eparis how's this? We've used this variable elsewhere to ensure that they don't apply configuration variables that we consider to be unsupported.